boms/cloud-lts-bom/pom.xml

```diff
@@ -288,7 +288,6 @@
       </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
-        <!-- BOM? -->
         <artifactId>google-cloud-bigquerystorage-bom</artifactId>
```

Comment:
@stephaniewang526 Can I use this BOM to cover API surface?

Reply:
Yes.